src/main/options.js

```diff
@@ -114,10 +116,59 @@ function getPlugin(options) {
   return printerPlugin;
 }
 
+function getInterpreter(filepath) {
+  if (typeof filepath !== "string") {
+    return "";
+  }
+
+  let fd;
+  try {
+    fd = fs.openSync(filepath, "r");
+  } catch (err) {
+    return "";
+  }
+
+  try {
+    const liner = new readlines(fd);
+    const firstLine = liner
+      .next()
+      .toString("utf8")
+      .trim();
```

Comment:
Shebangs should only occurred at the start of the line, this `trim` breaks it. Could you add a test for it as well?